src/complex-list-field/complex-list-field.component.scss

```diff
@@ -7,7 +7,7 @@
 .top-bar-container {
   width: 100%;
   position: sticky;
-  top: 0;
+  top: 46px;
```

Comment:
why `46px`

Reply:
https://inspirebeta.net/editor/holdingpen/12181

I want to stick the search bar just below the nav-bar in the references tab. So we should set it to the height of nav-bar,